elastic/kibana
# [Security Solution] [Bugfix] Fix analyzer missing from alert table

## Summary

The `graphOverlay` prop render got [accidentally deleted](https://github.com/elastic/kibana/pull/108644/files#diff-5796dde9b49a93dcce15ee9ea2133002123cbb9802033e07ae9db0c9739d08c6L337) from the integrated tgrid, which renders the Analyzer in the Alerts View. This caused a bug when clicking `...` context menu on an alert and selecting "Analyze", the analyzer would not show

I re-put the  `graphOverlay` prop render where it was, wrote tests, and removed some dead code

## To Test

- use siem-dev for server
- go to alerts page
- enter `agent.type: endpoint` in kql bar
- click the `...` context menu on an alert and select "Analyze"
- the analyzer should open... tada!

### Checklist

Delete any items that are not applicable to this PR.
